pkg/helpers/tokencmd/request_token_test.go

```diff
@@ -378,18 +357,6 @@ func TestRequestToken(t *testing.T) {
 			},
 			ExpectedToken: successfulToken,
 		},
-		"failing negotiate+prompting basic handler, negotiate+basic challenge, success": {
```

Comment:
Why is this test removed, vs changed to expected error

Reply:
updated, added that case back in w/ best guess, ptal 
was going to ask @stlaz about that as i'm a bit confused about negotiate challenge flow/forgot to ping him b4 he left today

Reply:
updated